src/components/GetApi.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
+import '../styles/app.css'
+import Loader from '../assets/steam-loading.gif'
+
+const GetApi = () => {
+  const [moviesData, setMoviesData] = useState([]);
+  const [loading, setLoading] = useState(false);
```

Comment:
Good work with the moviesData & loading stat. However, I don't see any error state